code/src/App.js

```diff
@@ -1,9 +1,33 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { Messages } from './components/Messages'
+import { SendMessage } from './components/SendMessage'
+import { Heart } from './img/Heart'
+
+// FONTAWESOME ICONS
+
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { faHeart, faSpinner, faQuoteRight, faQuoteLeft } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+library.add(faHeart, faSpinner, faQuoteLeft, faQuoteRight)
+
+// APP CONSTRUCTION
 
 export const App = () => {
+
+  const [messages, setMessages] = useState([])
+  const [loading, setLoading] = useState(true)
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <>
+      <header><h1>happy thoughts</h1><Heart /></header>
```

Comment:
Maybe this part can be in different lines so it's easier to read?
```suggestion
      <header>
        <h1>happy thoughts</h1>
        <Heart />
      </header>
```